packages/hwloc/hwloc.spec

```diff
@@ -0,0 +1,78 @@
+Name: %{_cross_os}hwloc
+Version: 2.12.2
+Release: 1%{?dist}
+Summary: Portable hardware locality library
+URL: http://www.open-mpi.org/projects/hwloc/
+License: BSD-2-Clause
+Source0: https://download.open-mpi.org/release/hwloc/v2.12/hwloc-%{version}.tar.bz2
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}systemd-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: hwloc development libraries and headers
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n hwloc-%{version} -p1
+
+%build
+%cross_configure \
+    --disable-cairo \
+    --disable-gl \
+    --disable-libudev \
+    --disable-libxml2 \
+    --disable-opencl \
+    --disable-pci \
+    --disable-plugins \
+    --enable-static \
+    --disable-shared \
+    --exec-prefix=%{_cross_prefix} \
+    --program-prefix=""
+
+%force_disable_rpath
+
+%make_build
+
+%install
+%make_install
+
+find %{buildroot} -name '*.la' -exec rm -f {} ';'
+find %{buildroot} -name '*.debug' -exec rm -f {} ';'
+
+%files
+%license COPYING
+%{_cross_attribution_file}
+%{_cross_bindir}/hwloc-annotate
+%{_cross_bindir}/hwloc-compress-dir
+%{_cross_bindir}/hwloc-gather-topology
+%{_cross_bindir}/hwloc-ls
+%{_cross_bindir}/hwloc-ps
+%{_cross_bindir}/hwloc-bind
+%{_cross_bindir}/hwloc-calc
+%{_cross_bindir}/hwloc-diff
+%{_cross_bindir}/hwloc-distrib
+%{_cross_bindir}/hwloc-info
+%{_cross_bindir}/hwloc-patch
+%{_cross_bindir}/lstopo
+%{_cross_bindir}/lstopo-no-graphics
```

Comment:
These are all binaries, not helper scripts?

Can we omit any of them?

Reply:
The primary use case I've seen is using `hwloc-ls` for confirming topology. I think `hwloc-info` might be similar. I added a reply to another comment but I'm not able to find the specific invocation we need for any specific binary so I think we may just keep this as a package that can be included in `-dev` variants. Either way, I think we can drop `lstopo` which is graphically oriented. Outside of the ls/info, the rest are for manipulating the topology files which would be useful if trying to debug this manually but honestly I think this would not be generally how things are done for containers.